performance.md

```diff
@@ -0,0 +1,341 @@
+---
+title: Performance Tuning and Tips
+---
+
+Most (if not all) Common Lisp implementations translates the source code into
```

Comment:
Perhaps "Many Common Lisp implementations..." ; I think CLISP is all byte code rather than assembly/machine instructions.

Reply:
small thing: "translate" rather than "translates"

Reply:
> I think CLISP is all byte code rather than assembly/machine instructions.
You are right. `disassemble` returns byte-code instructions instead of machine code.